workflows/templates/kebechet-run-results.yaml

```diff
@@ -0,0 +1,169 @@
+    app: thoth
+    component: kebechet-run-results
+spec:
+  entrypoint: kebechet-run-results
+  templates:
+    - name: kebechet-run-results
+      inputs:
+        parameters:
+          - name: "WORKFLOW_NAME"
+          - name: "THOTH_DOCUMENT_ID"
+          - name: "THOTH_ADVISER_METADATA"
+          - name: "IMAGE_STREAM_REGISTRY"
+          - name: "IMAGE_STREAM_NAMESPACE"
+          - name: "IMAGE_STREAM_TAG"
+      steps:
+        - - name: extract-metadata
+            template: extract-metadata
+            arguments:
+              parameters:
+                - name: "THOTH_ADVISER_METADATA"
+                  value: "{{inputs.parameters.WORKFLOW_NAME}}"
+                - name: "IMAGE_STREAM_REGISTRY"
+                  value: "{{inputs.parameters.WORKFLOW_NAME}}"
+                - name: "IMAGE_STREAM_NAMESPACE"
+                  value: "{{inputs.parameters.WORKFLOW_NAME}}"
+                - name: "IMAGE_STREAM_TAG"
+                  value: "{{inputs.parameters.WORKFLOW_NAME}}"
+        - - name: schedule-kebechet
+            template: schedule-kebechet
+            arguments:
+              parameters:
+                - name: "WORKFLOW_NAME"
+                  value: "{{inputs.parameters.WORKFLOW_NAME}}"
+                - name: "THOTH_DOCUMENT_ID"
+                  value: "{{inputs.parameters.WORKFLOW_NAME}}"
+                - name: "THOTH_ADVISER_METADATA"
+                  value: "{{inputs.parameters.WORKFLOW_NAME}}"
+                - name: "IMAGE_STREAM_REGISTRY"
+                  value: "{{inputs.parameters.WORKFLOW_NAME}}"
+                - name: "IMAGE_STREAM_NAMESPACE"
+                  value: "{{inputs.parameters.WORKFLOW_NAME}}"
+                - name: "IMAGE_STREAM_TAG"
+                  value: "{{inputs.parameters.WORKFLOW_NAME}}"
+                - name: "origin"
+                  value: "{{steps.extract-metadata.outputs.parameters.origin}}"
+                - name: "source_type"
+                  value: "{{steps.extract-metadata.outputs.parameters.source_type}}"
+            when: "{{steps.extract-metadata.outputs.parameters.source_type}} == kebechet"
+    - name: extract-metadata
+      inputs:
+        parameters:
+          - name: "THOTH_ADVISER_METADATA"
+          - name: "IMAGE_STREAM_REGISTRY"
+          - name: "IMAGE_STREAM_NAMESPACE"
+          - name: "IMAGE_STREAM_TAG"
+      script:
+        command:
+          - python3
+        name: extract-metadata
+        image: "{{inputs.parameters.IMAGE_STREAM_REGISTRY}}/\
+        {{inputs.parameters.IMAGE_STREAM_NAMESPACE}}/\
+        thoth-toolbox:{{inputs.parameters.IMAGE_STREAM_TAG}}"
+        source: |
+          import json
+
+          from thoth.common import init_logging
+
+          init_logging()
+
+          metadata = json.loads('{{inputs.parameters.THOTH_ADVISER_METADATA}}')
+
+          with open('/tmp/source_type', 'w') as f:
+            if metadata['source_type'] is not None:
+              f.write(metadata['source_type'])
+
```

Comment:
what about using `workflow-helpers`? :)

Reply:
Does thoth-toolbox have workflow-helpers installed now?

Reply:
https://quay.io/repository/thoth-station/workflow-helpers?tab=tags

Reply:
Do those images have all of this in it? https://github.com/thoth-station/workflow-helpers/pull/2

Reply:
if you do /deploy a new image will be created, we will use all this in the new cluster, so you can remove 
```
          - name: "IMAGE_STREAM_REGISTRY"
          - name: "IMAGE_STREAM_NAMESPACE"
          - name: "IMAGE_STREAM_TAG"
```
as we use images from the namespace, therefore `image: workflow-helpers`